### DISCUSSION TITLE
I haven't use Whisper for 2 month and I got issue (There is no help in GOOGLE)

### DISCUSSION THREAD

**Discussion by MrVertero** [openai/whisper#2143] [Q&A] [answered]
Hello guys, after not using Whisper for 2 months, I got some issue.

The transcription is running well but at the end I **don't have** any text generated...
Error:
```
[03:37.000 --> 03:40.000]  Ulica do codziesznej wody wrastoważy szansa.
Traceback (most recent call last):
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\Scripts\whisper.exe\__main__.py", line 7, in <module>
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\lib\site-packages\whisper\transcribe.py", line 457, in cli
    writer(result, audio_path, writer_args)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\lib\site-packages\whisper\utils.py", line 254, in write_all
    writer(result, file, options)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python39\lib\site-packages\whisper\utils.py", line 84, in __call__
    with open(output_path, "w", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: '.\\test.txt'

C:\Users\PC\Videos\Praca\Sesje Rady Rabka-Zdrój\Aktualne\Rabka lipiec (TEST)>
```

The key is _(I guess)_: _FileNotFoundError: [Errno 2] No such file or directory:_ **'.\test.txt'** 

When I google it people have problem that after **"No such file or directory:"** they have **ffmpeg**, but I think my works well (I do reinstall), and there is nothing writte with **ffmpego** but  **'.\test.txt'.**

**Comment by Erythros** [comment]
Cam here for the same reason. Similar situation, same problem.
Currently on Python 3.10.11

Not sure what else has been updated, but I assume something got updated and doesn't work anymore.

**Reply by glangford** [reply]
If you are using whisper from the command line, try setting the output directory to an absolute pathname you have write permissions for, using 
`--output_dir`

**Reply by MrVertero** [reply, marked as answer]
I managed to solve this problem.
You CAN'T have ANY spaces in the name of a file, folder, etc.
basically that's it

**Reply by Erythros** [reply]
That's whay I've been using. Here's an example:

```
whisper audio1.mp3 --model medium --output_format txt --output_dir /d/Programming/Documentation/Transcribes
```

Do you see anything wrong with this?
I reinstalled whisper, I reinstalled all versions for Python, ffmpeg.... nothing seems to work.

**Reply by MrVertero** [reply]
Are you going to cmd through folder where you have audio1?
You can also try step by step adding command for example:
1st try: whisper audio1.mp3 --model medium
2st try: whisper audio1.mp3 --model medium --output_format txt
![obraz_2024-05-06_130315155](https://github.com/openai/whisper/assets/150543754/f4f406b7-eae7-42e4-9e23-05e3bea505c0)
